docs/reference/ilm/actions/ilm-shrink.asciidoc

```diff
@@ -103,3 +91,25 @@ PUT _ilm/policy/my_policy
   }
 }
 --------------------------------------------------
+
+[[ilm-shrink-retry-failed-allocation]]
+==== Retrying failed allocation steps
+
+During a `shrink` action, {ilm-init} allocates the source index's primary shards
+to one node. After shrinking the index, {ilm-init} reallocates the shrunken
+index's shards to the appropriate nodes based on your allocation rules. These
+allocation steps can fail if:
+
+* A node is removed during the `shrink` action.
+* No node has enough disk space to host the source index's shards.
+* {es} cannot reallocate the shrunken index due to conflicting allocation rules.
```

Comment:
@jrodewig these are some examples of the possible conditions that might not be fulfilled. Shall we rephrase this a bit to illustrate that? (as opposed to being the only possible issues)

Reply:
Thanks @andreidan. I rephrased as follows:
> These allocation steps can fail for a number of reasons, including: